lib/tldr.js

```diff
@@ -29,17 +34,33 @@ class Tldr {
       });
   }
 
+  /**
+   * Print all pages in the cache.
+   * @param singleColumn {boolean} A boolean to print one command per line.
+   * @returns {Promise<void>} A promise when the operation is completed.
+   */
   listAll(singleColumn) {
     return index.commands()
       .then((commands) => {
         return this.printPages(commands, singleColumn);
       });
   }
 
+  /**
+   * Print a command page.
+   * @param commands {string[]} A given command to be printed.
```

Comment:
"A given command" is incorrect when we are passing an array. Let's change this to plural.